# On using batch-effect-corrected data for differential expressionanalysis of single-cell RNA sequencing data

| **Mouse Cell Atlas dataset** | **Human Pancreas dataset** |
| --- | --- |
| <img src="data/mca_dgsp2095_b155_13.png" width="350"> | <img src="data/pan_b13_dgsp2095_b155_13.png" width="350"> |

### Comparison of sparsity level distribution over cells  
<img src="data/sparsity_distribution_over_cells.png" width="450"> 

### data 
Including codes and results of processing 2 real datasets: **Mouse Cell Atlas** and **Human Pancreas**.

### results
Including code and processed data to quickly re-plot the submitted figures.

### code
This github includes **modified and additional code** for supporting the response to the article at: https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1850-9

The original code can be found at: https://github.com/JinmiaoChenLab/Batch-effect-removal-benchmarking
